README.md

```diff
@@ -176,25 +176,27 @@ The whole configuration is contained in file `config/{prod|dev|test}.exs` depend
 ### RESTful API configuration
 
 ```elixir
-config :maru, MongoosePush.Router,
-    versioning: [
-        using: :path
+config :mongoose_push, MongoosePushWeb.Endpoint,
+  https: [
```

Comment:
In this section we show example configuration, using `{:system, ...}` tuples is very confusing because those are not any specific values - this is part of configuration framework, that isn't even active until you build release.